subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -55,13 +55,13 @@ inline uint16_t ull_adv_handle_get(struct ll_adv_set *adv);
 
 static int init_reset(void);
 static inline struct ll_adv_set *is_disabled_get(uint8_t handle);
-static void ticker_cb(uint32_t ticks_at_expire, uint32_t remainder, uint16_t lazy,
-		      void *param);
+static void ticker_cb(uint32_t ticks_at_expire, uint32_t remainder,
+	uint16_t lazy, void *param);
```

Comment:
Use leading tabs, and then spaces to align with parameters.

Reply:
(See previous comment.)